### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by denyeart**
Go to 1.22
CouchDB to 3.3.3
Fabric to 2.5.7
Fabric-CA to 1.5.10

### CODE REVIEW

**Comment by denyeart** [comment]
@bestbeforetoday The java chaincode lint failure here is new. This is the first test execution since the java chaincode 2.5.2 release last week, could that be the cause?

**Comment by bestbeforetoday** [comment]
The `org.hyperledger.fabric-chaincode-java:fabric-chaincode-shim:2.5.2` release does seem to trigger this failure, although I don't obviously spot anything that changed in the chaincode-java code to cause this. It seems to be an issue with the ancient version of Mockito (5.5 years old) being used in this specific chaincode sample that fails. Updating to a newer version of Mockito makes everything happy again.

I have opened PR #1208 to update the test dependencies, which should fix this error. It solves it for locally at least.

**Comment by denyeart** [comment]
@Mergifyio rebase

**Review by satota2 — CHANGES_REQUESTED** [review]
@denyeart Thank you for the PR.
This is really a minor point, but the following line should be updated as well.
https://github.com/hyperledger/fabric-samples/blob/d3e2a90ad585f5a9903217b0286f55afc83414f4/test-network/scripts/utils.sh#L20-L21
https://github.com/hyperledger/fabric-samples/blob/d3e2a90ad585f5a9903217b0286f55afc83414f4/test-network/scripts/utils.sh#L162-L163

I think updating with another PR is an option. Please consider it.

**Comment by denyeart** [comment]
@satota2 Thanks for the feedback, done, I agree it is good to keep the versions updated in the same pull request so that we can easily reference them later for future updates.

File: .github/actions/test-network-setup/action.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ description: Set up the Test Network Runtime
 inputs:
   go-version:
     description: Version of go
-    default: '1.21'
+    default: '1.22'
```
Comment (pfi79) [inline .github/actions/test-network-setup/action.yaml:6]: maybe 1.22.3?

File: .github/workflows/lint.yaml [hunk 2]
```diff
@@ -16,7 +16,7 @@ concurrency:
   cancel-in-progress: true
 
 env:
-  GO_VER: '1.21'
+  GO_VER: '1.22'
```
Comment (pfi79) [inline .github/workflows/lint.yaml:19]: maybe 1.22.3

Comment (bestbeforetoday) [reply .github/workflows/lint.yaml:19]: Why would you want to restrict the Go version to a specific patch release and not allow the most recent patch release to be selected?

Comment (pfi79) [reply .github/workflows/lint.yaml:19]: If I read the documentation for go correctly, that's the way it's supposed to be now.

https://tip.golang.org/doc/go1.21

`
Go 1.21 introduces a small change to the numbering of releases. In the past, we used Go 1.N to refer to both the overall Go language version and release family as well as the first release in that family. Starting in Go 1.21, the first release is now Go 1.N.0. Today we are releasing both the Go 1.21 language and its initial implementation, the Go 1.21.0 release. These notes refer to “Go 1.21”; tools like go version will report “go1.21.0” (until you upgrade to Go 1.21.1). See “[Go versions](https://tip.golang.org/doc/toolchain#version)” in the “Go Toolchains” documentation for details about the new version numbering.
`

Comment (bestbeforetoday) [reply .github/workflows/lint.yaml:19]: This value is being to configure the setup-go action: https://github.com/actions/setup-go

Comment (pfi79) [reply .github/workflows/lint.yaml:19]: ok. Then don't.

**Review by satota2 — APPROVED** [review]
@denyeart Thank you for updating. LGTM!

**Pull request merged**